docs/HowTo/Get-Started/Connect-To-Testnet.md

```diff
@@ -0,0 +1,184 @@
+* Scope: single-client testnet (Lighthouse)
+* Spec version: `v0.10.1`
+* Teku version: [`v0.10.0`](https://github.com/PegaSysEng/teku/releases/tag/0.10.0)
+
+Lighthouse is an ETH2 client written in Rust. The _Testnet 5_ is the public single-client testnet mainly used by Lighthouse developers and users. It implements version `v0.10.1` of the ETH2 specification.
+
+It's possible to connect a Teku beacon-chain node to the Testnet 5 and synchronize the chain. Connecting Teku to Testnet 5 requires the pinned version [`v0.10.0`](https://github.com/PegaSysEng/teku/releases/tag/0.10.0) of Teku. Any version after the `v0.10.0` tag targets an updated version of the ETH2 spec and won't be able to connect to the Testnet 5.
+
+Teku does not contain a preset for Testnet 5. The necessary configuration can be found in the [eth2-testnets repository on Github](https://github.com/eth2-clients/eth2-testnets/tree/master/lighthouse/testnet5).
+
+## Enable an ETH1 Connection
+
+**Prerequisites**:
+
+* [Install Hyperledger Besu](https://besu.hyperledger.org/en/latest/HowTo/Get-Started/Install-Binaries) Ethereum 1.0 client in addition to the Teku Ethereum 2.0 client.
+
+The deposit contract for Schlesi and Topaz are deployed to the _Goerli_ ETH1 testing network. To enable an ETH1 connection on Teku, configure Hyperledger Besu to connect to the Goerli testnet and expose the RPC-HTTP APIs:
+
+```bash
+besu --network=goerli \
+--rpc-http-enabled=true \
+--rpc-http-port=8545 \
+--rpc-http-api=ETH,NET,WEB3
+```
+
+Once you have a local Goerli node running with an activated RPC-HTTP API, you can connect Teku to it by enabling `--eth1-enabled`.
+
+!!! example
+
+    ```
+    teku --network=schlesi --eth1-enabled --eth1-endpoint=localhost:8545
+    ```
+
+!!! tip
+
+    See also: [Hyperledger Besu - Run a node on Goerli testnet](https://besu.hyperledger.org/en/stable/HowTo/Get-Started/Starting-node/#run-a-node-on-goerli-testnet)
+
+## Boostrap a new Testnet
+
+If you already have created a custom testnet or plan to do so, Teku allows you to pass different configurations to the beacon-chain node. A custom network requires a couple of things:
+
+* A network configuration that can be either one of the `minimal` or `mainnet` presets, or a custom `yaml` file containing the network specification.
+* An `ssz` genesis state or information about the ETH1 deposit contract.
+* Eventually bootnodes to allow other users to connect to your network.
+
+**Network configuration**:
+
+The network configuration can be either of the presets `minimal` and `mainnet` or a `yaml` file. The custome configuration file can be passed as local file or remote config fetched via HTTP, e.g.:
+
+* `--network="minimal"`
+* `--network="mainnet"`
+* `--network="./path/to/chain.yaml"`
+* `--network="https://github.com/goerli/schlesi/raw/master/teku/chain.yaml"`
+
+!!! example
+
+    ```
+    teku --network="https://github.com/goerli/schlesi/raw/master/teku/chain.yaml"
+    ```
+
+**Deposit Contract**:
```

Comment:
Fix title case to sentence case. Only the first word and proper nouns should start with a capital letter.
You also have to do it in other titles, as for example, "testnet" is not a proper noun or trademark and should be lowercased.